python/taichi/ui/utils.py

```diff
@@ -107,7 +107,7 @@ def check_ggui_availability():
         wheel_tag = try_get_wheel_tag(taichi)
         if platform.system(
         ) == "Linux" and wheel_tag and 'manylinux2014' in wheel_tag:
-            libc_ver = try_get_loaded_libc_version()
```

Comment:
btw not sure if this is a typo, but if it is we probably want to fix it as well?

Reply:
Sorry it's a typo!